File: call-management/rule-management/get-call-recording-custom-greeting-list/code-samples/listCallRecordingCustomGreetings.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string accountId = "<ENTER VALUE>";

// OPTIONAL QUERY PARAMETERS
ListCallRecordingCustomGreetingsParameters listCallRecordingCustomGreetingsParameters = new ListCallRecordingCustomGreetingsParameters {
    //type = 'StartRecording'
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Account(accountId).CallRecording().CustomGreetings().List(listCallRecordingCustomGreetingsParameters);
// PROCESS RESPONSE

File: call-management/rule-management/get-call-recording-custom-greeting-list/code-samples/listCallRecordingCustomGreetings.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';

// OPTIONAL QUERY PARAMETERS
$queryParams = array(
    //'type' => 'StartRecording'
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->get("/restapi/v1.0/account/{$accountId}/call-recording/custom-greetings", $queryParams);
// PROCESS RESPONSE
?>
